ovos_config/config.py

```
@@ -387,22 +329,37 @@ def clear_cache(message=None):
         Configuration.updated(message)
 
 
+def update_assistant_config(config, bus=None):
+    f""" updates assistant config file with the contents of provided dict 
+    path: {ASSISTANT_CONFIG}
+    """
```

Ruff (0.8.2)

f-string used as docstring. Python will interpret this as a joined string, rather than a docstring.

(B021)